Extensions/TiledSpriteObject/Extension.cpp

```diff
@@ -29,6 +29,47 @@ void DeclareTiledSpriteObjectExtension(gd::PlatformExtension& extension) {
 #if defined(GD_IDE_ONLY)
   obj.SetIncludeFile("TiledSpriteObject/TiledSpriteObject.h");
 
+  obj.AddCondition("GetOpacity",
```

Comment:
Can we name this "Opacity"? New convention is "XXX" for conditions (no Get), "SetXXX" for actions, "XXX" for expressions (like conditions).